File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/client-cover-option.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';

export class ClientCoverOption extends BaseClass {
    clientCoverId: number;
    productOptionCoverId: number;
    numberOfMembers: number;
    premium: number;
    productOptionName: string;
    isMainFamily: boolean;
    totalPerLine: number;
    productOptionId: number;
}

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Policy/ClientCoverOption.cs
using RMA.Common.Entities;

namespace RMA.Service.ClientCare.Contracts.Entities.Policy
{
    public class ClientCoverOption : AuditDetails
    {
        public int ClientCoverId { get; set; }
        public int ProductOptionCoverId { get; set; }
        public int? NumberOfMembers { get; set; }
        public decimal? Premium { get; set; }
        public bool IsMainFamily { get; set; }
        public string ProductOptionName { get; set; }
        public decimal TotalPerLine { get; set; }
        public int ProductOptionId { get; set; }
    }
}